sw/device/boot_rom/BUILD

```diff
@@ -0,0 +1,78 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU", "opentitan_binary")
+load("//rules:autogen.bzl", "autogen_chip_info")
+
+autogen_chip_info(
+    name = "chip_info",
+)
+
+opentitan_binary(
+    name = "boot_rom",
+    linkopts = [
+        "-nostartfiles",
+        "-nostdlib",
+    ],
+    output_scrambled = True,
+    deps = [
+        ":boot_rom_lib",
+    ],
+)
+
+cc_library(
+    name = "boot_rom_lib",
+    srcs = [
+        "boot_rom.c",
+    ],
+    linkopts = [
+        #"-L $(location)",
+        "-T $(location rom_link.ld)",
+        #"--build-id=none",
```

Comment:
I don't think these commented items are required for anything.

Reply:
I agree, I'll remove them